src/rpc/client/page.ts

```diff
@@ -72,6 +74,7 @@ export class Page extends ChannelOwner<PageChannel, PageInitializer> {
     this.accessibility = new Accessibility(this._channel);
     this.keyboard = new Keyboard(this._channel);
     this.mouse = new Mouse(this._channel);
+    this.coverage = new Coverage(this._channel);
```

Comment:
does this mean feature detection no longer works here?

Reply:
I'll follow up.